docs/networking.md

```diff
@@ -1,6 +1,6 @@
 # How to use networking
 
-cloud-hypervisor can emulate one or more virtual network interfaces, represented at the hypervisor host by [tap devices](https://www.kernel.org/doc/Documentation/networking/tuntap.txt"). This guide briefly describes, in a manual and distribution neutral way, how to setup and use networking with cloud-hypevisor.
+cloud-hypervisor can emulate one or more virtual network interfaces, represented at the hypervisor host by [tap devices](https://www.kernel.org/doc/Documentation/networking/tuntap.txt). This guide briefly describes, in a manual and distribution neutral way, how to setup and use networking with cloud-hypevisor.
```

Comment:
Could you please edit your commit message as the following:
```
docs: networking: Fix typo in hyperlink

An extra double quote was introduced by mistake in one of the hyperlink.
```

Reply:
done

Reply:
@drmint80 Thanks for the fix and the update, and sorry in advance if I sound like a PITA... but what @sboeuf expected is for the git commit to be modified that way, not the github PR message. Also, we require commits to be SOB'ed, could you please sign it as well?
See https://github.com/intel/cloud-hypervisor/blob/master/CONTRIBUTING.md for a little more details.

Reply:
Yes sorry I was unclear, I should have pointed to the contributing documentation :)

Reply:
I added SOB tag, could you please check?

Reply:
@drmint80 you changed the github PR message, not the git one. You need to `git commit --amend`, add your SOB and change the commit message to HEAD, then git force push.

I tried to do that myself but I don't have the permissions to force push on your cloud-hypervisor master branch.

Reply:
Done